packages/ui/src/components/Chip.tsx

```diff
@@ -1,4 +1,4 @@
-import * as classNames from "classnames";
+import classNames from "classnames";
```

Comment:
@carcruz  can you clarify what is the issue with this import? In the link component it is imported in the same manner, should it be changed there as well?

https://github.com/opentargets/ot-ui-apps/blob/main/apps/genetics/src/components/Link.tsx#L4

Reply:
I remember changing this import because this error appeared in my IDE:

![image](https://github.com/opentargets/ot-ui-apps/assets/22417165/1bb7bcf2-64d2-450b-b983-967d96b4f15d)

Reply:
Hello @raskolnikov-rodion

In local development this is not firing an issue, but as you can see [here](https://main--ot-platform.netlify.app/target/ENSG00000282608), the build version of the app is not working do the change because if you do `import * as classNames from 'classNames'` the result of the classNames variable will be an object, instead of function. 
This is not your fault, we were missing a typescript configuration `"allowSyntheticDefaultImports": true` to not flag the import as a problem

Reply:
Hi @carcruz , thanks for the explanation and for providing the fix, indeed we can see the error I was looking for appearing in the built version. Bugs suck, but you're able to find the missing config so at least something good came out of it. If something similar happens in the future, please feel free to [reach me](mailto:oliveira_gus@hotmail.com) and I can help investigate 🙂

Reply:
Hi @carcruz , thanks for the explanation and for providing the fix, indeed we can see the error I was looking for appearing in the built version. Bugs suck, but you're able to find the missing config so at least something good came out of it. If something similar happens in the future, please feel free to [reach me](mailto:oliveira_gus@hotmail.com) and I can help investigate 🙂